recipes/onnx/all/conanfile.py

```diff
@@ -129,7 +129,7 @@ def _onnx_components(self):
             "libonnx": {
                 "target": "onnx",
                 "libs": ["onnx"],
-                "defines": ["ONNX_NAMESPACE=onnx", "ONNX_ML=1"],
+                "defines": ["ONNX_NAMESPACE=onnx", "ONNX_ML=1", "__STDC_FORMAT_MACROS"],
```

Comment:
But is it a private or public definition?

Reply:
So it's a public definition: https://github.com/onnx/onnx/blob/v1.11.0/onnx/version_converter/adapters/transformers.h#L7

Indeed `transformers.h` is a public header. But since header has been added in 1.11.0, so it would be neater to add this definition if version >= 1.11.0 only.